### PR TITLE
add biconomy aa support to eth passkeys example

### PR DESCRIPTION
**Pull request by andrewkmin**
## Summary & Motivation
$title

see changelog

## How I Tested These Changes
- Local

## Did you add a changeset?

If updating one of our packages, you'll likely need to add a changeset to your PR. To do so, run [`pnpm changeset`](https://pnpm.io/using-changesets#adding-new-changesets). `pnpm changeset` will generate a file where you should write a human friendly message about the changes. Note how this ([example](https://github.com/tkhq/sdk/blob/b409cd06790f011bf939adcf0755499b8e7497ae/.changeset/extra-http-exports.md?plain=1#L1)) includes the package name (should be auto added by the command) along with the type of [semver change (major.minor.patch)](https://semver.org/) (which you should set).

These changes will be used at release time to determine what packages to publish and how to bump their version. For more context see [this comment](https://github.com/tkhq/sdk/pull/67#issuecomment-1568838440).

### CODE REVIEW

File: examples/with-eth-passkeys-galore/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # @turnkey/with-eth-passkey-signers
 
+## 0.2.0
```
Comment (andrewkmin) [inline examples/with-eth-passkeys-galore/CHANGELOG.md:3]: this is primarily for tracking on our side. think a changelog would be helpful in keeping track of changes over time

File: examples/with-eth-passkeys-galore/src/pages/index.tsx [hunk 2]
```diff
@@ -178,6 +278,94 @@ export default function Home() {
...
   };
 
+  const signTransactionWithAAClient = async (
+    data: signTransactionFormData,
+    getSmartAccount: (
+      data: signTransactionFormData
+    ) => Promise<BiconomySmartAccountV2>,
+    options?: { nonceOptions?: { nonceKey: number } }
+  ) => {
+    const { destinationAddress, amount } = data;
```
Comment (omkarshanbhag) [inline examples/with-eth-passkeys-galore/src/pages/index.tsx:288]: tiny nit: do we want to do any input validation?

File: examples/with-eth-passkeys-galore/README.md [hunk 3]
```diff
@@ -28,13 +28,20 @@ $ pnpm install # Install dependencies
 $ pnpm run build  # Compile source code
 ```
 
-### 2/ Setting up Turnkey
```
Comment (omkarshanbhag) [inline examples/with-eth-passkeys-galore/README.md:31]: Would be great to be more explicit about the two options of how to run this example in the README!! 

It looks like we either do an AA based flow or a normal ETH signing flow based on whether or not  `NEXT_PUBLIC_BICONOMY_BUNDLER_URL` and `NEXT_PUBLIC_BICONOMY_PAYMASTER_API_KEY` are set.

Comment (omkarshanbhag) [reply examples/with-eth-passkeys-galore/README.md:31]: Also a quick line on what Biconomy is and what functionality it adds to this example

**Review by omkarshanbhag — APPROVED** [review]
LGTM -- Just some requests for further context and nits

File: examples/with-eth-passkeys-galore/src/pages/index.tsx [hunk 4]
```diff
@@ -16,10 +23,15 @@ type subOrgFormData = {
   subOrgName: string;
 };
 
-type signingFormData = {
+type signMessageFormData = {
   messageToSign: string;
 };
 
+type signTransactionFormData = {
```
Comment (taylorjdawson) [inline examples/with-eth-passkeys-galore/src/pages/index.tsx:30]: nit: should we capitalize these types?

File: examples/with-eth-passkeys-galore/src/pages/index.tsx [hunk 5]
```diff
@@ -366,6 +560,53 @@ export default function Home() {
           </form>
         </div>
       )}
+      {wallet && (
+        <div>
+          <h2>... and now let&apos;s sign a transaction!</h2>
```
Comment (taylorjdawson) [inline examples/with-eth-passkeys-galore/src/pages/index.tsx:565]: maybe a little more readable
```suggestion
          <h2>... and now let{"'"}s sign a transaction!</h2>
```

or even 
```
          <h2>... and now {"let's"} sign a transaction!</h2>
```

**Review by taylorjdawson — APPROVED** [review]
Approved with nits! 🙌

File: examples/with-eth-passkeys-galore/src/pages/index.tsx (hunk 2 above)
Comment (andrewkmin) [reply examples/with-eth-passkeys-galore/src/pages/index.tsx:288]: we should in a more serious application but will leave it out of this PR for now :)

**Review by omkarshanbhag — APPROVED** [review]
lfg

**Pull request merged**